BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1842,6 +1886,9 @@ void EndBoost()
 
         IEnumerator CruiseRoutine()
         {
+            burnRate = (sustainerMass > 0) ? (sustainerMass / cruiseTime) * Time.fixedDeltaTime : 0;
+            ordinanceMass = (boosterMass > 0) ? boosterMass : 0;
+            float massToBurn = (boosterMass > 0) ? boosterMass + sustainerMass : sustainerMass;
             StartCruise();
```

Comment:
If you wanted to optimize, you could stick these three fields inside a ``if (useFuel)`` block to prevent them from being calced when !useFuel.